library/src/morpheus-master/js/src/sakai.morpheus.i18n.utils.js

```diff
@@ -1,6 +1,6 @@
 (function ($) {
 
-    if (!portal.loggedIn){
+    if (!portal.loggedIn || !(portal.bullhorns && portal.bullhorns.enabled)) {
```

Comment:
Same here I believe this is no longer needed.

Reply:
same ;)